src/c++/perf_analyzer/command_line_parser.cc

```diff
@@ -994,10 +1034,16 @@ CLParser::ParseCommandLine(int argc, char** argv)
         }
         break;
       }
-      case 20:
-        params_->using_custom_intervals = true;
-        params_->request_intervals_file = optarg;
+      case 20: {
+        std::string request_intervals_file{optarg};
+        if (IsFile(request_intervals_file)) {
+          params_->request_intervals_file = request_intervals_file;
+          params_->using_custom_intervals = true;
+        } else {
+          Usage("--request-intervals must be a valid file path");
```

Comment:
I like how you put the actual option name in the error here. It is much easier for them to figure out what is wrong and what needs to change. Can you update the rest of the cases to be like that? Putting the option name instead of english text describing what the option does?

Reply:
Sure, I changed the descriptions accordingly.